synapse/lib/modelrev.py

```diff
@@ -64,6 +65,43 @@ async def save():
                 if nodeedits:
                     await save()
 
+    async def revModel20211112(self, layers):
+        # uniq and sort several array types
+        todoprops = (
+            'biz:rfp:requirements',
+
+            'crypto:x509:cert:ext:sans',
+            'crypto:x509:cert:ext:crls',
+            'crypto:x509:cert:identities:fqdns',
+            'crypto:x509:cert:identities:emails',
+            'crypto:x509:cert:identities:ipv4s',
+            'crypto:x509:cert:identities:ipv6s',
+            'crypto:x509:cert:identities:urls',
+            'crypto:x509:cert:crl:urls',
+
+            'inet:email:message:headers',
```

Comment:
Since they can be duplicate within the protocol should we maintain their order and their ability to be duplicate here?

Reply:
Removed in latest push.